internal/pkg/cli/dialog/use_template.go

```diff
@@ -5,120 +5,185 @@ import (
 	"fmt"
 	"strconv"
 
+	"github.com/keboola/keboola-as-code/internal/pkg/cli/options"
 	"github.com/keboola/keboola-as-code/internal/pkg/cli/prompt"
+	"github.com/keboola/keboola-as-code/internal/pkg/project"
 	"github.com/keboola/keboola-as-code/internal/pkg/template/input"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils"
+	useTemplate "github.com/keboola/keboola-as-code/pkg/lib/operation/project/local/template/use"
+	loadState "github.com/keboola/keboola-as-code/pkg/lib/operation/state/load"
 )
 
 type contextKey string
 
-func (p *Dialogs) AskUseTemplateOptions(inputs *input.Inputs) (results map[string]interface{}, err error) {
-	results = make(map[string]interface{})
-	ctx := context.Background()
-	errors := utils.NewMultiError()
-	for _, i := range inputs.All() {
-		if !i.Available(results) {
+type useTmplDialogDeps interface {
+	Options() *options.Options
+	ProjectState(loadOptions loadState.Options) (*project.State, error)
+}
+
+type useTmplDialog struct {
+	*Dialogs
+	deps             useTmplDialogDeps
+	loadStateOptions loadState.Options
+	out              useTemplate.Options
+	context          context.Context        // for input.ValidateUserInput
+	inputsValues     map[string]interface{} // for input.Available
```

Comment:
Dokoncil som `use template` dialog:
- generuje sa priamo `out - useTemplate.Options` - nie map.
- hodnota inputu sa uklada na 3 miesta: 
  - `context` - pre  `input.ValidateUserInput` (to uz bolo)
  - `inputsValues` - pre `for input.Available`
  - a do `out`
Aby nebolo vela duplicitneho kodu, tak to riesi metoda `addInputValue`.